miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -1082,6 +1082,9 @@ end
 #! Where:
 #! - block_height_delta is the desired expiration time delta (1 to 0xFFFF).
 export.update_expiration_block_num
```

Comment:
We should update line 1079 above to be:
```
Inputs: [KERNEL_PROCEDURE_HASH, block_height_delta, ...]
```

Reply:
I have a question regarding these two procedures. Some of the other procedures in this file (for example `get_note_serial_number`), after dropping the procedure hash and executing the respective internal procedure, also drop some padding. Should these new procedures also drop padding? If so, how much padding should be dropped? Some procedures (like `get_note_serial_number`) drop a whole word, but others (like `get_fungible_faucet_total_issuance`) only drop an element.

Reply:
hmmm - good question! I think procedures in this file should not really care about dropping padding since `exec_kernel_proc` handles this generically. But this may be just a remnant of the old approach which should be fixed comprehensively once we close #685.